1208-projects/project_landing_page.feature

```diff
@@ -16,6 +16,17 @@ Feature: Project Landing Page
             | Scientific summary |
             | Participants       |
             | Results            |
+        And they see number of content of Participants and Results
+
+    @xxx
+    Scenario: User with access rights opens Landing Page for Project
+    Given User opens Landing Page for Project
+    And User is listed as Participant with the Role of Project Manager on project
+    Then they can see a Edit button
+    And they can see a Delete button
```

Comment:
Dette blir kanskje litt rart siden `When` egnetlig ikke er en action, men heller en `And`, men tror det er påkrevd å bruke When. @Eiriknil bør verifisere dette

```suggestion
    Scenario: Project Manager opens Landing Page for Project
        Given User opens Landing Page for Project
        When the User is Project Manager for the Project
        Then they can see an Edit button
        And they can see a Delete button
```

Reply:
Bytt teksten i `Given`og `When`så gir det mer mening.
```suggestion
    Scenario: User with access rights opens Landing Page for Project
    Given User is a Participant with the Role of Project Manager on a project
    When User opens Landing Page for Project
    Then they can see a Edit button
    And they can see a Delete button
```

Reply:
Da får vi ikke gjenbrukt andre Scenario i Given da, men det er kanskje greit?

Reply:
Fleire av endringene i denne PR omhandler Given - eg har slavisk brukt andre scanerio i Given. Har oppfattet at det var normen.

Reply:
OK, var litt ensidig fokusert på dette scenario, ignorer forslaget mitt :)

Reply:
Jeg har etter beste evne prøvd å gjøre det samme